tests/integration/multi-arch.bash

```diff
@@ -1,13 +1,22 @@
 #!/bin/bash
+
 get_busybox() {
+	local sha
+
 	case $(go env GOARCH) in
 	arm64)
-		echo 'https://github.com/docker-library/busybox/raw/dist-arm64v8/glibc/busybox.tar.xz'
+		# To update to the latest package, copy top commit id from
+		# https://github.com/docker-library/busybox/commits/dist-arm64v8?author=docker-library-bot
+		sha=385dcf62178190a3cc8ef5129a3236466d3abab5
 		;;
 	*)
```

Comment:
* -> amd64

Reply:
That's the way it was, I am not changing anything here. Seems that amd64 is the default.

We can certainly error out here on an unknown arch.